### ISSUE TITLE
error: unexpected condition for llvm::BranchInst

### ISSUE THREAD

**Issue by practicalswift** [NASA-SW-VnV/ikos#57]
```
$ cat testcase.c
int a() { return a ? 0 % 0 ?: 0 : 0; }
$ clang-7 -c testcase.c -o /dev/null 2> /dev/null
$ echo $?
0
$ ikos testcase.c
[*] Compiling testcase.c
…
[*] Running ikos preprocessor
[*] Running ikos analyzer
[*] Translating LLVM bitcode to AR
ikos-analyzer: /tmp/ikos-zjFW68/testcase.pp.bc: error: unexpected condition for llvm::BranchInst
ikos: error: a run-time error occurred
```

**Comment by arthaud** [comment]
Fixed in eb1503a2768c2d128bc6f3c9810fcfa07b9ea414.

By the way, IKOS doesn't report any error here because clang optimized `0 % 0` into `undef`.
That's why it's important to first look at the compiler warnings before using IKOS.

**Comment by practicalswift** [comment]
Confirming that the fix did solve the problem. Thanks for the quick fix!
